htdocs/mri/jiv/get_file.php

```diff
@@ -92,58 +132,68 @@
     exit(4);
 }
 
+$PathPrefix = '';
 switch($FileExt) {
 case 'mnc':
-    $FullPath         = $mincPath . '/' . $File;
+    $PathPrefix       = $mincPath;
```

Comment:
Well I did verify it works and fixes the security issue (which has been confirmed by @MounaSafiHarab). You're right this is more of a cosmetic change but it also makes the file a little smaller and I think clearer.